app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -82,6 +82,9 @@ def set_derived_fields!
         self.prevten = 30 if owning_organisation&.provider_type == "LA"
       end
     end
+    if is_bedsit? && form.start_year_after_2024?
+      self.beds = 1
+    end
```

Comment:
Would it make sense to group all the 2024 things together under a single if statement?

Reply:
Sure, I'll nest this for future use